config:
  Filter_Para: 0.1
  Normal_Radius: 8.0
  Lidar_filter: 0.1
  grid_resolution: 2.0
  Box_x: 0.9
  Box_y: 0.8
  Box_z: 1.1
  fov_distance: 4.0
  Fov_Min_Dis: 2.0